omnibridge_api/app/data/ai_models_metadata.json

```diff
@@ -3,12 +3,16 @@
         "id": 1, 
         "name": "ChatGPT", 
         "type": "chat_bot",
-        "description": "Ask any question"
+        "description": "Ask any question",
+        "input_dtype": "text",
+        "output_dtype": "text"
```

Comment:
do we need these fields for the UI?

Reply:
Yes, i"m creating generic components that would be applied to different models. 
For example, text to text, is a component with 2 text fields
text to image, is a component with text field and image placeholder.